handwritten_support/RiscvExtras.lean

```diff
@@ -0,0 +1,136 @@
+
+import THE_MODULE_NAME.Sail.Sail
```

Comment:
This breaks the build for me. How is `The_MODULE_NAME` supposed to be updated?

Reply:
Are you on MacOS or Linux? the lean translator uses `sed` to replace it with the actual name of the module. Until [the PR fixing sed on linux](https://github.com/rems-project/sail/pull/1041) is merged it's not going to work :/

Reply:
My CI runs on Linux. I meanwhile fixed it with my own sed call in the CI.

Reply:
Indeed, this works now and this comment is resolved.